README.md

```diff
@@ -157,13 +188,44 @@ Leave that connection up and running in the background and move on to the next s
 
 ## Connecting to DOOM
 
+### Use noVNC (browser)
+
+You can connect to ocpdoom via a browser - no software install!
+
+Using `oc` we build a container that contains `noVNC` with environment variables (ENDPOINT, PORT, PASSWORD) that connect to ocpdoom service exposed above.
+
+```bash
+# create noVNC container
+oc new-app https://github.com/codekow/container-novnc.git \
+  -n ocpdoom \
+  --name novnc \
+  -e ENDPOINT='ocpdoom' \
+  -e PORT='5900' \
+  -e PASSWORD='openshift'
+```
+
+We then expose the novnc service, or create a route, that allows ingress via tls on port 443 into OpenShift.
+
+```bash
+# create route
+oc expose service \
+  -n ocpdoom \
+  novnc \
+  --overrides='{"spec":{"tls":{"termination":"edge"}}}'
+
+route=$(oc -n ocpdoom get route novnc -o jsonpath='{.spec.host}')
+
+echo "https://${route}"

```

Comment:
When testing against https the application is not available. Only http works.

Reply:
I know it's kinda obvious via the command but maybe mention it again explicitly that they user needs to login with the password given.

Reply:
> When testing against https the application is not available. Only http works.

`--overrides='{"spec":{"tls":{"termination":"edge"}}}'` should enable https. 

I'm not sure what is failing. This worked for me?